### ISSUE TITLE
Perforamnce of Kurapan EAST 

### ISSUE THREAD

**Issue by pragazv** [kurapan/EAST#27]
I tried to execute the prediction code of argman east and kurapan east . Accuracy doesn't vary much rather performance vary a lot. For example with a "X" CPU configuration, argman EAST took on an avg - 0.4 to 0.5 sec to do prediction where as Kurapan EAST is consuming on an average 60 sec or more for an image on the same machine
is this because of tf.slim vs tf.keras?  or is there anything else which delay the prediction
please give me the clarity.

**Comment by kurapan** [comment]
Sorry for a late response. 
The speed of my implementation is similar to argman's TF implementation. 
Are you sure that you run it on a gpu? If you use Keras 2.1.5 and TF 1.8, it should work and be as fast as argman's implementation. How do you evaluate the performance?

Btw. I don't use tf.keras. I use regular Keras and TF for backend.

**Comment by KeshriPooja** [comment]
I am getting 7 to 8 sec processing time for east argman model how can i reduce processing time.
